### ISSUE TITLE
[Bug]: IoCExtract classifies a FQDN as EMAIL

### ISSUE THREAD

**Issue by giovino** [microsoft/msticpy#681]
**Describe the bug**

When using `IoCExtract()` on a string of a FQDN, I expect it to classify the string as a `dns`.

**To Reproduce**

```
>>> from msticpy.transform import IoCExtract
>>> ioc_extractor = IoCExtract()
>>> iocs = ioc_extractor.extract('news.ycombinator.com')
>>> print(iocs)
defaultdict(<class 'set'>, {'email': {'news.ycombinator.com'}})
```

**Expected behavior**

I expect IoCExtract to extract the string `news.ycombinator.com` as `dns`. 

**Environment**

- Python Version: 3.11.4
- OS: MacOS 12.6.7
- Python environment: Local Python using pyenv. 
- MSTICPy Version: 2.5.3

**Comment by ianhelle** [comment]
Urgh - good spot.
I think it's matching a defanged version of an email address, interpreted as news.ycombinATor.com.
Not sure that this defanging method is used very widely (as opposed to defanging just the domain section) so the simplest thing might be to remove the match on "AT"

**Comment by ianhelle** [comment]
This should be fixed in #684 and will go out in 2.7.0 release
